src/test/java/net/masterthought/cucumber/generators/EscapeHtmlReferenceTest.java

```diff
@@ -0,0 +1,34 @@
+package net.masterthought.cucumber.generators;
+
+import org.apache.velocity.app.event.ReferenceInsertionEventHandler;
+import org.junit.Test;
+
+import static org.apache.commons.lang.StringEscapeUtils.escapeHtml;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author M.P. Korstanje (mpkorstanje@github)
+ */
+public class EscapeHtmlReferenceTest {
+
+    private static final String SOME_REFERENCE = "someReference";
+    private final ReferenceInsertionEventHandler insertionEventHandler = new EscapeHtmlReference();
+
+    @Test
+    public void referenceInsert_returnNormalText(){
+        String normalText = "a plain statement";
+        assertThat(insertionEventHandler.referenceInsert(SOME_REFERENCE, normalText)).isEqualTo(normalText);
```

Comment:
I prefer https://en.wikipedia.org/wiki/Behavior-driven_development as the way how tests are implemented